docs/public/using/gui/README.md

```diff
@@ -0,0 +1,5 @@
+# Using the GUI
+
+* The [GUI API][API]
```

Comment:
Eventually, we should describe how to use Zonemaster’s GUI here, possibly with screenshots. I didn’t understand why this document mentioned (another) API until I read it, and understood that said API is not really an application programming interface, but just a collection of URL tricks for populating some of the inputs.

Reply:
A pointer to a work from OVH to provide such documentation: https://github.com/ovh/docs/pull/3471